src/App.tsx

```diff
@@ -17,18 +36,32 @@
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                setSelectedFilter={setSelectedFilter}
```

Comment:
The `setSelectedFilter` and `setQuery` methods do not follow the naming conventions for event handler functions. According to the conventions, handler names should start with 'handle' and include the name of the event. For example, `setSelectedFilter` could be renamed to `handleFilterChange`. Here is a link to the article that explains this in detail: [here](https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364)